src/ceph.in

```diff
@@ -183,6 +183,67 @@ def mdsids():
         l.append(mdsdict['name'])
     return l
 
+def mgrids():
+    ret, outbuf, outs = json_command(cluster_handle, prefix='mgr dump',
+                                     argdict={'format': 'json'})
+    if ret == -errno.EINVAL:
+        # try old mon
+        ret, outbuf, outs = send_command(cluster_handle,
```

Comment:
i don't think we need to support server not supporting json commands, we use json to represent command parameters back to hammer.